app/User.php

```diff
@@ -4,20 +4,22 @@
 
 use Illuminate\Support\Facades\Hash;
 use Illuminate\Notifications\Notifiable;
+use Illuminate\Database\Eloquent\SoftDeletes;
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 
 class User extends Authenticatable
 {
     use Notifiable;
+    use SoftDeletes;
 
     /**
      * The attributes that are mass assignable.
      *
      * @var array
      */
     protected $fillable = [
-        'first_name', 'last_name', 'email', 'password', 'avatar_path',
+        'first_name', 'last_name', 'email', 'password', 'avatar_path', 'admin_flag',
     ];
```

Comment:
When array values become many better to format it like this, this is also suggested in th PSR standards
```
    protected $fillable = [
        'first_name',
        'last_name',
        'email',
        'password',
        'avatar_path',
        'admin_flag',
    ];
```

Reply:
I formated these values.